openedx/core/djangoapps/user_api/preferences/api.py

```diff
@@ -392,6 +394,18 @@ def validate_user_preference_serializer(serializer, preference_key, preference_v
                 "user_message": user_message,
             }
         })
+    if preference_key == "time_zone" and preference_value not in common_timezones_set:
+        developer_message = ugettext_noop(u"Value '{preference_value}' not valid for preference '{preference_key}': "
+                                          u"Not in timezone set.")
```

Comment:
I think you need to make this a single string literal, and mark it with `# pylint: disable=line-too-long` so the translation works properly.